cluster/examples/kubernetes/ceph/replicated-storageclass.yaml

```diff
@@ -6,11 +6,6 @@ metadata:
 spec:
```

Comment:
What if we left this file named `storageclass.yaml`? I know I suggested the longer name, but I don't like my own suggestion as much now. :) It would make the common sample simpler.